tests/cypress/integration/actions_organizations/case_113_new_organization_pipeline.js

```diff
@@ -0,0 +1,276 @@
+        });
+
+        it('Open the organization settings. Invite members.', () => {
+            cy.openOrganization(organizationParams.shortName);
+            cy.checkOrganizationParams(organizationParams);
+            cy.checkOrganizationMembers(1, [firstUserName]);
+            const membersToInvite = [
+                {
+                    email: capitalizeEmail(secondUser.emailAddr),
+                    role: 'Worker',
+                },
+                {
+                    email: thirdUser.emailAddr,
+                    role: 'Worker',
+                },
+            ];
+            cy.inviteMembersToOrganization(membersToInvite);
+            cy.checkOrganizationMembers(3, [firstUserName, secondUserName, thirdUserName]);
+        });
+
+        it('Create a project, create a task. Deactivate organization.', () => {
+            cy.goToProjectsList();
+            cy.createProjects(
+                project.name,
+                project.label,
+                project.attrName,
+                project.attrVaue,
+                project.multiAttrParams,
+            );
+            cy.goToTaskList();
+            cy.createAnnotationTask(taskName, labelName, attrName, textDefaultValue, archiveName);
+            cy.openTask(taskName);
+            cy.assignTaskToUser(secondUserName);
+            cy.deactivateOrganization();
+        });
+
+        // Activate after implementation
+        it.skip('The project, the task are invisible now.', () => {
+            cy.contains('.cvat-item-task-name', taskName).should('not.exist');
+            cy.goToProjectsList();
+            cy.contains('.cvat-projects-project-item-title', project.name).should('not.exist');
+        });
+
+        it('Admin tries to leave from organization (not successfully because he is not a member of it).', () => {
+            cy.logout(firstUserName);
+            cy.login();
+            cy.activateOrganization(organizationParams.shortName);
+            cy.openOrganization(organizationParams.shortName);
+            cy.contains('button', 'Leave organization').should('be.visible').click();
+            cy.get('.cvat-modal-organization-leave-confirm')
+                .should('be.visible')
+                .within(() => {
+                    cy.contains('button', 'Leave').click();
+                });
+            cy.get('.cvat-notification-notice-leave-organization-failed').should('exist');
+            cy.closeNotification('.cvat-notification-notice-leave-organization-failed');
+        });
+
+        it('Second user login. The user is able to see the organization, the task and can\'t see the project.', () => {
+            cy.logout();
+            cy.login(secondUserName, secondUser.password);
+            cy.checkOrganizationExists(organizationParams.shortName);
+            cy.contains('.cvat-item-task-name', taskName).should('exist');
+            cy.goToProjectsList();
+            cy.get('.cvat-empty-projects-list').should('exist');
```

Comment:
A comment about this case:
Need to activate organization first. The second user cannot see the task if do not switch organization context (now the user sees the task, but it is a bug, will be fixed soon). The second user also sees the project (when organization is activated) on projects page (this is one more bug, will be fixed)

Reply:
Fix for the second one here: https://github.com/openvinotoolkit/cvat/pull/4201

Reply:
I merged last changed from ```develop``` with fix #4201. The second bug is not preproducing. I also thought it would be good to separate some tests and add additional steps. Please take a look.
I have commented out the checks that are not working now.